addons/library.kodi.guilib/libKODI_guilib.h

```diff
@@ -825,13 +825,13 @@ class CAddonGUIRenderingControl
   virtual ~CAddonGUIRenderingControl();
   virtual void Init();
 
-  virtual bool Create(int x, int y, int w, int h, void *device);
+  virtual bool Create(int x, int y, int w, int h, float ratio);
```

Comment:
how about adding a second method with an additional parameter "ratio" that returns false if not implemented. We call this first and if it fails the current one. This way we could maintain backwards compatibility until addons have been updated. thoughts?